core/libp2p/muxer/muxed_connection_config.hpp

```diff
@@ -7,17 +7,37 @@
 #define KAGOME_MUXED_CONNECTION_CONFIG_HPP
 
 #include <cstdint>
+#include <cstddef>
 
 namespace libp2p::muxer {
   /**
    * Config of muxed connection
    */
   struct MuxedConnectionConfig {
+   public:
+    static constexpr uint32_t kDefaultMaximumWindowSize = 1024u * 1024u;
+    static constexpr uint32_t kDefaultMaximumStreams = 1000u;
+
+    /**
+     * @brief construcsts configuration instance
+     * @param max_window_size how much unconsumed data each stream can
+     have
+     * stored locally
+     * @param max_streams how much streams can be supported by Yamux at
+     one time
+     */
+    inline MuxedConnectionConfig(uint32_t max_window_size, uint32_t max_streams)
+        : maximum_window_size{max_window_size}, maximum_streams{max_streams} {}
+
+    inline MuxedConnectionConfig()
```

Comment:
No need to write inline in class members - they are inline by default

Reply:
what about `one definition rule`?